[2024-05-25T21:44:01.910+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-25T21:44:01.932+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: DAG_catedra_airflow.first-step-el scheduled__2024-05-25T21:43:00+00:00 [queued]>
[2024-05-25T21:44:01.939+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: DAG_catedra_airflow.first-step-el scheduled__2024-05-25T21:43:00+00:00 [queued]>
[2024-05-25T21:44:01.940+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-05-25T21:44:01.952+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): first-step-el> on 2024-05-25 21:43:00+00:00
[2024-05-25T21:44:01.959+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=1241) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-25T21:44:01.960+0000] {standard_task_runner.py:63} INFO - Started process 1243 to run task
[2024-05-25T21:44:01.961+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'DAG_catedra_***', 'first-step-el', 'scheduled__2024-05-25T21:43:00+00:00', '--job-id', '254', '--raw', '--subdir', 'DAGS_FOLDER/dag_clase.py', '--cfg-path', '/tmp/tmpsopzyhr5']
[2024-05-25T21:44:01.962+0000] {standard_task_runner.py:91} INFO - Job 254: Subtask first-step-el
[2024-05-25T21:44:02.001+0000] {task_command.py:426} INFO - Running <TaskInstance: DAG_catedra_airflow.first-step-el scheduled__2024-05-25T21:43:00+00:00 [running]> on host b2cfee8f0b2b
[2024-05-25T21:44:02.085+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='DAG_catedra_***' AIRFLOW_CTX_TASK_ID='first-step-el' AIRFLOW_CTX_EXECUTION_DATE='2024-05-25T21:43:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-05-25T21:43:00+00:00'
[2024-05-25T21:44:02.086+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-25T21:44:02.097+0000] {logging_mixin.py:188} INFO - Connection to database was succesful
[2024-05-25T21:44:02.111+0000] {logging_mixin.py:188} INFO - Query completed
[2024-05-25T21:44:02.114+0000] {logging_mixin.py:188} INFO - execute_query_result_set()
[2024-05-25T21:44:02.121+0000] {logging_mixin.py:188} INFO - Connection to database was succesful
[2024-05-25T21:44:02.133+0000] {logging_mixin.py:188} INFO - Inserts performed wihout fail
[2024-05-25T21:44:02.134+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-05-25T21:44:02.134+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-25T21:44:02.158+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=DAG_catedra_***, task_id=first-step-el, run_id=scheduled__2024-05-25T21:43:00+00:00, execution_date=20240525T214300, start_date=20240525T214401, end_date=20240525T214402
[2024-05-25T21:44:02.214+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-25T21:44:02.233+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-05-25T21:44:02.237+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
